### ISSUE TITLE
Error While executing Set-AzureRmRecoveryServicesAsrVaultContext -Vault $vault

### ISSUE THREAD

**Issue by shubhamgupta514** [Azure/azure-powershell#5648]
@viverm  @markcowl 

I am getting below error.Please look into this.

Set-AzureRmRecoveryServicesAsrVaultContext : Cannot bind parameter 'Vault'. Cannot convert the
"Microsoft.Azure.Commands.RecoveryServices.ARSVault" value of type
"Microsoft.Azure.Commands.RecoveryServices.ARSVault" to type "Microsoft.Azure.Commands.RecoveryServices.ARSVault".

I am using below module versions currently.

4.0.2      AzureRM.RecoveryServices
0.2.1      AzureRM.RecoveryServices.SiteRecovery

**Comment by viverm** [comment]
Use this to unblock 

$path = Get-AzureRmRecoveryServicesVaultSettingsFile -Vault $Vault
Import-AzureRmRecoveryServicesAsrVaultSettingsFile -Path $path.filepath

**Comment by maddieclayton** [comment]
Closing in favor of: https://github.com/Azure/azure-powershell/issues/5635, which has more detail and conversation.
